javascript/ql/src/Security/CWE-094/ExpressionInjection.qhelp

```diff
@@ -8,18 +8,20 @@
 			code injection in contexts like <i>run:</i> or <i>script:</i>.
 		</p>
         <p>
-            Code injection in GitHub Actions may allow an attacker to 
-            exfiltrate the temporary GitHub repository authorization token. 
+            Code injection in GitHub Actions may allow an attacker to
+            exfiltrate any secrets used in the workflow and
+            the temporary GitHub repository authorization token.
             The token might have write access to the repository, allowing an attacker
-             to use the token to make changes to the repository.
+            to use the token to make changes to the repository.
         </p>
 	</overview>
 
 	<recommendation>
 		<p>
 			The best practice to avoid code injection vulnerabilities
 			in GitHub workflows is to set the untrusted input value of the expression
-			to an intermediate environment variable.
+			to an intermediate environment variable and then use the environment variable
+			using the native syntax of the shell/script interpreter (i.e. <b>NOT</b> the <i>${{ env.VAR }}</i>).
```

Comment:
```suggestion
			using the native syntax of the shell/script interpreter (that is, not <i>${{ env.VAR }}</i>).
```
We try to avoid the "i.e." abbreviation as it is confusing to some users. We also avoid <b>SCREAMING CASE</b> and similar.